src/lxc/storage/lvm.c

```diff
@@ -177,10 +185,18 @@ static int do_lvm_create(const char *path, uint64_t size, const char *thinpool)
 	cmd_args.vg = vg;
 	cmd_args.lv = lv;
 	cmd_args.size = sz;
+	cmd_args.sigwipe = true;
 	TRACE("Creating new lvm storage volume \"%s\" on volume group \"%s\" "
 	      "of size \"%s\"", lv, vg, sz);
 	ret = run_command(cmd_output, sizeof(cmd_output),
 			  lvm_create_exec_wrapper, (void *)&cmd_args);
+
+	//If lvcreate is old and doesn't support signature wiping, try again without it.
```

Comment:
We only use `/* */` style comments in LXC. :) Would be nice if you could change it. :)

Reply:
@brauner got it, will do :)